### DISCUSSION TITLE
Beta macos dmg available?

### DISCUSSION THREAD

**Discussion by mtissington** [utmapp/UTM#5213] [Releases]
I'd like to install the latest beta on macos.

Is there a dmg available?
If so where ? (I maybe overlooking the obvious)

**Comment by tifasoftware** [comment]
If you go to the main page, and look for Releases, click on +93 Releases. The betas will be shown there.
May I ask what why would need a beta version out of curiosity?

**Comment by mtissington** [comment]
Clicking on +93 Releases - does show the details about the latest beta release but it does not seems to provide a download link for any platform.

As for why - I want to live on the edge :)

**Comment by mtissington** [comment]
Just found the installs - needed to click on Assets to expand the list ... thanks.
